README.md

```diff
@@ -549,29 +575,66 @@ A revocation message MUST conform to the following format:
 
 This format makes it easy to select the relevant UCAN, confirm that the issuer is in the proof chain for some or all capabilities, and validate that the revocation was signed by the same issuer.
 
-Any other witnesses in the selected UCAN not issued by the same DID as the revocation issuer MUST be treated as valid.
+Any other proofs in the selected UCAN not issued by the same DID as the revocation issuer MUST be treated as valid.
```

Comment:
I'm not sure I understand this. What are the other proofs ? I assume selected UCAN is the one addressed by the `ucanCid` and it's either issued by `did` or not. Maybe an example of token where revocation was partial would be helpful ?

I used to think that meant that if I encounter a UCAN with multiple proofs where some are revoked, ones that aren't are still vaild, if that is the case I think rephrasing would be a good idea as "selected" is confusing here.

Reply:
Here's the scenario:

You have a UCAN `A`. It has 3 proofs `B`, `C`, and `D`. `D` is revoked. You want to use a capability that depends on on `B` but not `D`. You should continue to be able to use that without having to ask for a reissued token. This is especially important if the revoked token is deep in a chain, and you may not have a direct relationship with the issuer.

Reply:
That was my understanding from before. I think term "selected" is bit confusing here though, because reading it now made me think of token been revoked and not the token that uses revoked token in proofs.

Reply:
Good feedback! Will clarify 👍 Maybe diagrams would be helpful, too.

Reply:
https://github.com/ucan-wg/spec/pull/57